Dockerfile

```diff
@@ -29,4 +29,4 @@ RUN chmod +x ./chainlink-launcher.sh
 
 EXPOSE 6688
 ENTRYPOINT ["./chainlink-launcher.sh"]
-CMD ["node"]
+CMD ["local node"]
```

Comment:
Small nitpick: let's change these to

```
CMD ["local", "node"]
```

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#cmd